storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -1193,6 +1199,12 @@
     public static final String SUPERVISOR_CPU_CAPACITY = "supervisor.cpu.capacity";
 
     /**
+     * A map of resources the Supervisor has e.g {"cpu" : 200.0. "memory.capacity.mb": 256.0, "gpu" : 0.5 }
```

Comment:
How does this compare to `supervisor.cpu.capacity`? or the memory capacity? What keys are valid?  More explanation here would be good.